src/runtime/entitlements-manager.js

```diff
@@ -854,6 +859,18 @@ export class EntitlementsManager {
           url = addQueryParam(url, 'sut', swgUserToken);
         }
 
+        // Add interaction_age param.
+        if (readTime) {
+          const last = parseInt(readTime, 10);
+          if (last) {
+            const now = Date.now();
+            const interaction_age = Math.floor((now - last) / 1000);
```

Comment:
I would just do the `Date.now()` directly inside. No need for the extra variable since it's only used on this line and doesn't provide any additional context

Reply:
Done.